test/validation.html

```diff
@@ -47,5 +48,23 @@
       expect(event.detail.valid).to.be.false;
     });
   });
+  describe('invalid cannot be set to false', () => {
+    let customField;
+  
+    beforeEach(async() => {
+      customField = fixture('custom-field');
+      customField._shouldSetInvalid = (invalid) => invalid;
+      await nextRender();
+    });
+  
+    it('should set invalid only when it is true', async() => {
+      customField.required = true;
+      customField.validate();
+      expect(customField.invalid).to.be.true;
+      customField.value = 'value';
```

Comment:
If we set only one sub-value here, let's also update the custom-field in the fixture so that it has `<input type="text">` only for the sake of consistency.